db/system_keyspace.hh

```diff
@@ -112,6 +112,7 @@ static constexpr auto LARGE_CELLS = "large_cells";
 static constexpr auto SCYLLA_LOCAL = "scylla_local";
 static constexpr auto RAFT = "raft";
 static constexpr auto RAFT_SNAPSHOTS = "raft_snapshots";
+static constexpr auto REPAIR = "repair";
```

Comment:
repair_history?

Reply:
If we want to add more repair related tables, yes, repair_history makes sense.

Reply:
even if we don't, it's better to have descriptive names. let's say now we think we won't add more repair related tables, but we're wrong, and we end up adding another one in the future, let's say a couple of years down the road. then we're stuck with repair table for compatibility reasons. so it's better to start with a descriptive name upfront.

Reply:
Yes, makes sense.